dcos/package.py

```diff
@@ -1384,6 +1384,31 @@ def latest_version(self):
         pkg_versions.sort()
         return pkg_versions[-1]
 
+    def get_package_revision(self, package_version=None):
```

Comment:
Off the cuff, maybe it would help to globally replace all mentions of "package version" with "package number" to disambiguate it from the version of the software in the package.  The original naming is kind of unfortunate.